### PR TITLE
categories_screen

### PR DESCRIPTION
**Pull request by shayb**
taks 2 with fixes of pr

### CODE REVIEW

File: app/src/main/java/org/kinecosystem/kinit/repository/TasksRepository.kt [hunk 1]
```diff
@@ -17,131 +17,6 @@ private const val TASK_STATE_KEY = "task_state"
 private const val SHOW_CAPTCHA_KEY = "show_captcha"
 
 class TasksRepository {
```
Comment (berryve) [inline app/src/main/java/org/kinecosystem/kinit/repository/TasksRepository.kt]: This file should be deleted

File: app/build.gradle [hunk 2]
```diff
@@ -65,6 +65,7 @@ android {
     testOptions {
         unitTests {
             includeAndroidResources = true
+            unitTests.returnDefaultValues = true
```
Comment (berryve) [inline app/build.gradle]: why is this needed ?

Comment (katzio) [reply app/build.gradle]: we don't, I remember removing it. It's a dangerous to use:
Caution: Setting the returnDefaultValues property to true should be done with care. The null/zero return values can introduce regressions in your tests, which are hard to debug and might allow failing tests to pass.
